Form/Type/OptionType.php

```diff
@@ -92,22 +100,11 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults(array(
             'data_class' => 'Hackzilla\Bundle\PasswordGeneratorBundle\Entity\Options',
             'csrf_protection' => false,
+            'generator' => null,
         ));
     }
 
-
-    /**
-     * {@inheritdoc}
-     */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
-    {
-        $this->configureOptions($resolver);
-    }
-
-    /**
-     * @return string
-     */
-    public function getName()
+    public function getBlockPrefix()
     {
         return '';
```

Comment:
The prefix was being added to the url parameters.

This was the only way I found to keep the parameters simple